js/app.js

```diff
@@ -0,0 +1,145 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Widać, że znasz tylko podstawy JS. Dobrze jakbyś się nauczyła prototypów albo klas. Wtedy by to wyglądało dużo lepiej.